testData/solutions/win32.json5

```diff
@@ -1587,1517 +2267,14267 @@
+                                "0x416",
+                                "0x809",
+                                "0x80a",
+                                "0xc0a",
+                                "0xc0c"
+                            ],
+                            "enumLabels": [
+                                "English - US",
+                                "German",
+                                "Finish",
+                                "French - France",
+                                "Italian",
+                                "Portuguese - Brazil",
+                                "English - United Kingdom",
+                                "Spanish - Mexico",
+                                "Spanish - Spain",
+                                "French - Canada"
+                            ],
+                            "default": "0x409"
+                        }
+                    },
+                    "PTB-Global.UppercasePitchDelta": {
+                        "schema": {
+                            "title": "Uppercase pitch delta",
+                            "description": "Pitch variation as you type or arrow accross capital letters.",
+                            "type": "integer",
+                            "minimum": -99,
+                            "maximum": 999,
+                            "default": 50
+                        }
+                    },
+                    "PTB-Global.Volume": {
+                        "schema": {
+                            "title": "Global Volume",
+                            "description": "The volume for all announcements.",
+                            "type": "integer",
+                            "minimum": 10,
+                            "maximum": 100,
+                            "default": 100
+                        }
+                    },
+                    "PTB-Global.VoiceName": {
+                        "schema": {
+                            "title": "Global voice name",
+                            "description": "The voice name to use by default.",
+                            "enum": [
+                                // If synth is eloq
+                                "Reed",
+                                "Shelly",
+                                "Bobby",
+                                "Rocko",
+                                "Glen",
+                                "Sandy",
+                                "Grandma",
+                                "Grandpa",
+                                // If synth is mmobile
+                                "Microsoft Daniel",
+                                "Microsoft Maria",
+                                // If synth is SAPI 5X 64
+                                "Microsoft Maria Desktop"
+                            ],
+                            "enumLabels": [
+                                // If synth is eloq
+                                "
```

Comment:
Sorry to have not caught these before, I only noticed these when starting to reconcile them with the google sheet.  IMO we should not onboard these.  The JAWS entry alone would be unmanageably large (973 settings!).